cdap-master/src/main/java/co/cask/cdap/master/environment/k8s/StorageMain.java

```diff
@@ -95,5 +90,7 @@ protected void configure() {
     // Create metadata tables
     injector.getInstance(MetadataStorage.class).createIndex();
     injector.getInstance(LevelDBTableService.class).close();
```

Comment:
is this still needed?

Reply:
Yes. Because we don't know what the config is, and calling it always is safe.